extension/lib/settings_view.js

```diff
@@ -22,16 +49,16 @@ class SettingsView {
         this.highlighterSetting = "Blue"; // str: "Blue", Yellow", or "Green"
         this.shadowSetting = "Blue"; // str: "Blue", "Yellow", or "Green"
         
+        // Setting these here instead of beginning of file bc need to wait due to async problems,
+        // window.trackerStyle is not ready if set in beginning of file, is ready if set here
```

Comment:
See comments here; if trackerStyle = window.trackerStyle was set in the beginning of the file, then (presumably due to async reasons), it was not set in time for the updateSettings() call, which needs it. This may be a little hacky of a fix for this, but it works. Added settings here too so that they're in the same place.

Reply:
Hmm i wanna think later to see if theres a better way, but i dont want to block u
can you leave a TODO: for now so we can revisit later?